tern/.htaccess

```diff
@@ -3,86 +3,98 @@ RewriteEngine On
 # Using 303 redirects as per https://www.w3.org/TR/swbp-vocab-pub/#recipe3
 
 
-######################################
-# https://w3id.org/tern/ontologies/org
-######################################
+#######################################
+# https://w3id.org/tern/ontologies/org/
+#######################################
 
 # Turtle
 RewriteCond %{QUERY_STRING} ^_format=text/turtle$ [OR]
 RewriteCond %{HTTP:Accept} text/turtle [NC]
+RewriteRule ^ontologies/org/$ https://raw.githack.com/ternaustralia/ontology_tern-org/master/docs/tern-org.ttl [R=303,L]
```

Comment:
You can simplify all these and use one rule that accepts both forms.  Just make the `/` optional:
```
RewriteRule ^ontologies/org/?$ https://...
```

Reply:
Thanks for the tip. I've removed the redundant rule and added the regex optional `?`.

Reply:
Merged, but then noticed this change and one below still have the redundant rule.  Feel free to change later, should work either way.